src/index.ts

```diff
@@ -61,6 +67,22 @@ export function form<Decl extends FormDeclaration<any>>(
     }: Sources<Decl>): Sinks<Decl> {
         const touchedKeys = new Set<keyof Decl>();
 
+        let customSubmission$: Stream<KeyboardEvent> = Stream.empty();
+        if (options.customSubmission) {
```

Comment:
Now that it has the default argument, `options.customSubmission` can never be falsy.

Is it intended to make it optional?
(Sorry, I've missed that in the last review.)

Reply:
You're right. Now I can rewrite this part like this:

```
const { customSubmission } = options;
const { fields: submissionFields } = customSubmission;
const { predicate } = customSubmission;
const customSubmission$ = Stream.merge(
  ...Array.from(submissionFields).map(key =>
    isolateSource(DOM, key).events('keydown').filter(predicate),
  ),
);
```

Reply:
Refactored: https://github.com/herp-inc/cyclic-form/pull/2/commits/51fae7a1d5ae0bbf59c92699229fce161a71d974